stdlib/REPL/src/latex_symbols.jl

```diff
@@ -211,27 +228,24 @@ const latex_symbols = Dict(
     "\\euro" => "€",
 
 # 732 symbols generated from unicode.xml
-    "\\textexclamdown" => "¡",
+    "\\exclamdown" => "¡",
     "\\sterling" => "£",
     "\\yen" => "¥",
-    "\\textbrokenbar" => "¦",
+    "\\brokenbar" => "¦",
     "\\S" => "§",
-    "\\textasciidieresis" => "¨",
     "\\copyright" => "©",
-    "\\textordfeminine" => "ª",
+    "\\ordfeminine" => "ª",
     "\\neg" => "¬",
     "\\circledR" => "®",
-    "\\textasciimacron" => "¯",
     "\\degree" => "°",
     "\\pm" => "±",
-    "\\textasciiacute" => "´",
     "\\P" => "¶",
     "\\cdotp" => "·",
-    "\\textordmasculine" => "º",
-    "\\textonequarter" => "¼",
-    "\\textonehalf" => "½",
-    "\\textthreequarters" => "¾",
-    "\\textquestiondown" => "¿",
+    "\\ordmasculine" => "º",
+    "\\onequarter" => "¼",
+    "\\onehalf" => "½",
+    "\\threequarters" => "¾",
+    "\\questiondown" => "¿",
     "\\AA" => "Å",
     "\\AE" => "Æ",
     "\\DH" => "Ð",
```

Comment:
Shouldn't this be named "Eth" (or "ETH") for consistency with `"\\eth" => "ð"` below?

Reply:
(I'd also add a comment, as was done for `\ltphi` below, to distinguish from `\DJ`)

Reply:
Seems like ð should instead be called `\dh` below to be consistent with [latex](http://hevea.inria.fr/examples/test/sym.html).  The `\eth` name comes from the AMS math symbols list, though.  So maybe we should just have both `\dh` and `\eth`.

Reply:
I added `\dh` as a synonym for `\eth`.